src/components/RichTextEditor/RichTextEditor.cy.tsx

```diff
@@ -539,3 +547,29 @@ describe('RichTextEditor Component', () => {
         );
     });
 });
+
+describe('RichTextEditor Component: Positioning of Toolbar', () => {
+    it('should render with floating toolbar', () => {
+        cy.mount(<RichTextWithToolbarPositioning />);
+
+        cy.get(TOOLBAR).should('not.visible');
```

Comment:
Could you use different test id's in https://github.com/Frontify/fondue/pull/925/files#diff-0c679c41e3945db40c4d0d07465fe4b3be96d1285ed475c4d9d779ae5cb5e184R8 to distinguish if the correct wrapper is rendered?

Reply:
Updated